Some Issue fixes

RaphaelIT7 opened the pull request:
[+] Added install.md to help users install Volt.
[#] Fixed some engine traces failing.
 (some traces the engine uses to determent if you can unduck failed) 
[#] Fixed CheckCollision checking collisions even if collisions are disabled(IsCollisionEnabled = false) 
[#] JoltPhysicsObject::GetName() now returns a proper name instead of "NoName" 
In Gmod If you used tostring(Entity(1):GetPhysicsObject()) it returned NoName instead of player_stand or player_crouch.
[#] Fixed a crash

Related issues:
fixes #128
fixes #164 
fixes #102
fixes #137
fixes #85
fixes #125 (Read the [install.md](https://github.com/Joshua-Ashton/VPhysics-Jolt/blob/2ee44fc1faaabc96f374c95d203f20168a74ccd0/install.md))

Over the next few days, I want to fix more issues.

RaphaelIT7 committed:
Small bug fixes

[#] Fixed some engine traces failing.
 (some traces the engine uses to determent if you can unduck failed)
[#] Fixed CheckCollision checking collisions even if collisions are disabled(IsCollisionEnabled = false)
[#] JoltPhysicsObject::GetName() now returns a proper name instead of "NoName"
In Gmod If you used tostring(Entity(1):GetPhysicsObject()) it returned NoName instead of player_stand or player_crouch.

Joshua-Ashton commented:
The reason the portal back exists right now is because with it enabled, walking up static props ends up with the player getting stuck sometimes.

RaphaelIT7 commented:
I will test a bit around with it.

RaphaelIT7 committed:
Crash fix + Added install.md

[+] Added install.md to help users install Volt.
[#] Fixed a crash
[#] Set m_bSimulating to true after m_ContactListener.PostSimulationFrame()

RaphaelIT7 committed:
Fixed install.md formatting.

Joshua-Ashton commented:
Can you split these fixes out into separate MRs?

RaphaelIT7 commented:
Every fix or some specific fixes?

Joshua-Ashton commented:
Please all of the fixes, or at least into separate commits.

RaphaelIT7 commented:
> Please all of the fixes, or at least into separate commits.
Will be done. Gimme a few min.

RaphaelIT7 commented:
Should be done. Should I make a separate PR for each future fix, or is it okay if they have their separate commit?
